src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+@import '//fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,100&display=swap';
```

Comment:
The import statement is incomplete and missing the file path to import. Make sure to specify the correct path to the SCSS file you want to import.